pkg/server/ca/manager/manager.go

```diff
@@ -782,6 +916,14 @@ func (u *bundleUpdater) SyncX509Roots(ctx context.Context, roots []*x509certific
 		})
 	}
 
+	// Notificate about tainted authorities to force rotation of intermediates,
+	// and update database,
+	// it is done in a separated thread to prevent agent and downstream servers,
+	// to start rotations before current server forced intermediate rotations
```

Comment:
```suggestion
	// Notificate about tainted authorities to force the rotation of
	// intermediates and update the database. This is done in a separate thread
	// to prevent agents and downstream servers to start the rotation before the
	// current server starts the rotation of the intermediate.
```